bin/Cargo.toml

```diff
@@ -65,6 +65,7 @@ cfg-if.workspace = true
 clap = { workspace = true, default-features = false, features = ["cargo", "derive", "help", "std", "suggestions"] }
 futures-util = { workspace = true, default-features = false, features = ["std"] }
 ipnet = { workspace = true, features = ["serde"] }
+libc.workspace = true
```

Comment:
Should this be limited to `target_family = "unix"`?

(`[target.'cfg(windows)'.dependencies]`)